ccd-definition/AuthorisationCaseField/CareSupervision/system-update.json

```diff
@@ -607,5 +607,82 @@
     "CaseFieldID": "reviewDecision1",
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "CRUD"
+  },
+  {
```

Comment:
Added this in to support e2e - 

`await I.submitNewCaseWithData(mandatorySubmissionFields);`

without adding system-update permissions the e2e test would fail in submitting new case.